lib/compiler.js

```diff
@@ -298,13 +294,13 @@ function findModelDefinitions(rootDir, sources) {
   return registry;
 }
 
-function resolveSourceDir(rootDir, sourceDir) {
-  var srcDir = path.resolve(rootDir, sourceDir);
+function resolveAppPath(rootDir, relativePath) {
+  var srcDir = path.resolve(rootDir, relativePath);
   if (fs.existsSync(srcDir))
     return srcDir;
 
   // Handle module-relative path, e.g. `loopback/common/models`
```

Comment:
This comment feels misleading now, because the code below handles both relative paths and module-relative paths now. I am proposing to move the comment to the "if" block below, and add a comment to the "else" block too.